<ui version="4.0" >
 <author>Niko Sams</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>VocabelCoach</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>&amp;Search:</string>
        </property>
        <property name="buddy" >
         <cstring>filterLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filterLineEdit" />
      </item>
      <item>
       <widget class="QComboBox" name="boxSelectionCombo" />
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="vocableEditorView" >
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
      <property name="selectionBehavior" >
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Edit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionAddVocable" />
    <addaction name="actionEditVocable" />
    <addaction name="actionDeleteVocable" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="actionSaveAs" />
    <addaction name="separator" />
    <addaction name="actionQuiz" />
    <addaction name="separator" />
    <addaction name="actionPrint" />
    <addaction name="separator" />
    <addaction name="actionImport" />
    <addaction name="actionExport" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menu_Edit" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionNew" />
   <addaction name="actionOpen" />
   <addaction name="actionSave" />
   <addaction name="separator" />
   <addaction name="actionAddVocable" />
   <addaction name="actionEditVocable" />
   <addaction name="actionDeleteVocable" />
   <addaction name="separator" />
   <addaction name="actionQuiz" />
  </widget>
  <widget class="QDockWidget" name="dockWidget" >
   <property name="windowTitle" >
    <string>Statistics</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>9</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QListView" name="listView" />
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew" >
   <property name="text" >
    <string>&amp;New</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="text" >
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>&amp;Save</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs" >
   <property name="text" >
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionImport" >
   <property name="text" >
    <string>&amp;Import...</string>
   </property>
  </action>
  <action name="actionPrint" >
   <property name="text" >
    <string>&amp;Print...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExport" >
   <property name="text" >
    <string>&amp;Export...</string>
   </property>
  </action>
  <action name="actionAddVocable" >
   <property name="text" >
    <string>Add Vocable</string>
   </property>
  </action>
  <action name="actionEditVocable" >
   <property name="text" >
    <string>Edit Vocable</string>
   </property>
  </action>
  <action name="actionDeleteVocable" >
   <property name="text" >
    <string>Delete Vocable</string>
   </property>
  </action>
  <action name="actionQuiz" >
   <property name="text" >
    <string>&amp;Quiz</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
